src/controller/routeController.js

```diff
@@ -0,0 +1,69 @@
+import model from '../database/models';
+
+// eslint-disable-next-line import/prefer-default-export
+
+export const addRoute = (req, res) => {
+  const rt = {
+    name: req.body.name,
+    orginID: req.body.origin,
+    destinationID: req.body.destination,
+    busStops: req.body.busStops
+  };
+
+  model.Route.create(rt)
+    .then((data) => {
+      res.status(201).json({ status: 201, message: 'Route created successfully', data });
+    })
+    .catch((err) => res.status(500).json({ status: 404, message: err }));
```

Comment:
Why do u return here two different status codes?

Reply:
I removed the second ones

Reply:
![error](https://user-images.githubusercontent.com/55746279/95309912-0b1fa900-088c-11eb-9332-42b123619832.JPG)
Marius can you bring back capital T in that file?